### ISSUE TITLE
Rename this project?

### ISSUE THREAD

**Issue by chrismlusk** [chrismlusk/accuraty-templates#61]
@Accuraty has brought it up multiple times: We might consider a different name for this project. I'm opening this issue so we can discuss.

**Comment by chrismlusk** [comment]
Initially, I was trying to build this to have a skin/theme that could work on multiple CMS. But that quickly dissolved, and this is now DNN-specific. Therefore, my current proposal is `DNN-starter`.

**Comment by Accuraty** [comment]
I still like AccuTheme :)

**Comment by chrismlusk** [comment]
Yeah, but this is more of a "starter kit" than a "theme" overall. It's a starting point for building themes (and other components) for DNN sites, but it's not a finished product like if you were downloading a WordPress theme.

That's why I like "starter" in the name because it makes it more explicit about what this is.

Maybe `AccuTheme-starter`?
